openpype/settings/entities/schemas/projects_schema/schemas/schema_maya_publish.json

```diff
@@ -129,6 +129,69 @@
             ]
         },
 
+        {
+            "type": "dict",
+            "collapsible": true,
+            "key": "ValidateMayaUnits",
+            "label": "Validate Maya Units",
+            "checkbox_key": "enabled",
+            "children": [
+                {
+                    "type": "boolean",
+                    "key": "enabled",
+                    "label": "Enabled"
+                },
+                 {
+                    "type": "boolean",
+                    "key": "optional",
+                    "label": "Optional"
+                },
+                {
+                    "type": "boolean",
+                    "key": "validate_linear_units",
+                    "label": "Validate linear units"
+                },
+                {
+                          "key": "linear_units",
+                          "label": "Linear units",
+                          "type": "enum",
+                          "multiselection": false,
+                          "defaults": "cm",
+                          "enum_items": [
+                            {"mm": "millimeter"},
+                            {"cm": "centimeter"},
+                            {"m":  "meter"},
+                            {"km":  "kilometer"},
+                            {"in":  "inch"},
```

Comment:
Why do some of these entries have two spaces before the value? Were these entered manually? Or does the settings tool do that?

Reply:
thats just probably my PyCharm forcing his idea of padding of json files.